types/applepayjs/index.d.ts

```diff
@@ -181,6 +181,35 @@ declare class ApplePaySession extends EventTarget {
     static readonly STATUS_PIN_REQUIRED: number;
 }
 
+/**
+ * A customizable error type that you create to indicate problems with the address or contact information on an Apple Pay sheet.
+ */
+declare class ApplePayError implements ApplePayJS.ApplePayError {
```

Comment:
Do we still need `ApplePayJS.ApplePayError`? It seems unnecessary now, though it would be a breaking change to remove it.